sdk/analytics/src/main/java/com/klaviyo/analytics/networking/requests/KlaviyoApiRequest.kt

```diff
@@ -315,6 +338,7 @@ internal open class KlaviyoApiRequest(
     protected open fun parseResponse(connection: HttpURLConnection): Status {
         // https://developers.klaviyo.com/en/docs/rate_limits_and_error_handling
         responseCode = connection.responseCode
+        responseHeaders = connection.headerFields
```

Comment:
btw, from what i can tell, the typing here is non-nullable already so it should be safe to make the type of `responseHeaders` non-nullable without needing to handle it